api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
```

Comment:
I tried to do that initially but I was getting that I was using a mutable default value as an argument.  The only way I know to get around that is to reset it every time which is a lot of extra lines.

![screen shot 2015-11-02 at 4 32 20 pm](https://cloud.githubusercontent.com/assets/9755598/10894900/19a250ca-817f-11e5-8536-7d0b9c7956dc.png)